test/v2-migration-tests/src/test/resources/software/amazon/awssdk/v2migrationtests/maven/before/src/main/java/foo/bar/S3.java

```diff
@@ -121,13 +121,18 @@ private void singleBucketArgMethods(AmazonS3 s3, String bucket) {
         s3.getBucketReplicationConfiguration(bucket);
         s3.getBucketTaggingConfiguration(bucket);
         s3.getBucketWebsiteConfiguration(bucket);
+        s3.disableRequesterPays(bucket);
+        s3.enableRequesterPays(bucket);
+        s3.isRequesterPaysEnabled(bucket);
     }
 
     private void bucketKeyArgsMethods(AmazonS3 s3, String bucket, String key) {
         s3.deleteObject(bucket, key);
         s3.getObject(bucket, key);
         s3.getObjectAcl(bucket, key);
         s3.getObjectMetadata(bucket, key);
+        s3.getObjectAsString(bucket, key);
```

Comment:
Can we define a variable for response and validate the response type is also updated?